server/storage/players.lua

```diff
@@ -369,6 +372,24 @@ RegisterCommand('convertjobs', function(source)
     TriggerEvent('qbx_core:server:jobsconverted')
 end, true)
 
+---Removes invalid groups from the player_groups table.
+RegisterCommand('cleanplayergroups', function(source)
+	if source ~= 0 then return warn('This command can only be executed using the server console.') end
+
+    local groups = MySQL.query.await('SELECT DISTINCT `group`, type FROM player_groups')
+    for i = 1, #groups do
+        local group = groups[i]
+        local validGroup = group.type == GroupType.JOB and GetJob(group.group) or GetGang(group.group)
+        if not validGroup then
+            MySQL.query.await('DELETE FROM player_groups WHERE `group` = ? AND type = ?', {group.group, group.type})
+            lib.print.info(('Remove invalid %s %s from player_groups table'):format(group.type, group.group))
+        end
+    end
+
+    lib.print.info('Removed invalid groups from player_groups table')
+    TriggerEvent('qbx_core:server:playergroupscleaned')
```

Comment:
What kind of action would be taken on this event?
```suggestion
    TriggerEvent('qbx_core:server:playerGroupsCleaned')
```

Reply:
Was just following the pattern above. probably not too useful

Reply:
Should we keep it then? If there's no use case, I would be in favor of removing the event.